### DISCUSSION TITLE
Internal time

### DISCUSSION THREAD

**Discussion by lvaoliveira** [serkri/SmartEVSE-3#55] [General]
How to adjust the internal clock?
Does it connect to ntp server?

Thank you

**Comment by dingo35** [comment]
Yes it connects through ntp server so there is not much to adjust...

**Reply by lvaoliveira** [reply]
ok.. so I think I found an issue, if you set a static ip, it will not get DNS servers, then ntp will not connect.
Also, how do I change it from static ip to DHCP back?

**Reply by dingo35** [reply]
If you configure a static IP, you should configure your gateway and netmask also. You might even need to configure your DNS manually because you are not using the DNS that DHCP is advertising.

You can get into the configuring portal by selecting "setup wifi" on the display.
